R/variable-as-methods.R

```diff
@@ -0,0 +1,143 @@
+#' as.* methods for variables
+#'
+#' Use the as.* family of functions to make a derived copy of a variable that 
+#' has been converted into a new type.
+#' 
+#' Each type of Crunch variable (text, numeric, categorical, etc.) has an `as.*`
+#' function (`as.Text`, `as.Numeric`, and `as.Categorical` respectively) that 
+#' takes the input given as `x`, and makes a new derived variable that is now of 
+#' the type specified. See below for detailed examples.
+#' 
+#' For `as.Text` and `as.Numeric`, aliases to the R-native functions 
+#' `as.character` and `as.numeric` are provided for convenience.
+#'
+#' @param x a Crunch variable to derive and convert to a new type
+#' @param format for `as.Datetime`, when the variable in `x` is a text or 
+#' categorical variable, the format that the date-time is formatted in (default: 
+#' \code{"\%Y-\%m-\%d \%H:\%M:\%S"}); for `as.Text` and `as.Categorical`, the 
+#' format that the datetime should be formatted as (default: 
+#' \code{"\%Y-\%m-\%d \%H:\%M:\%S"}).
+#' @param resolution for `as.Datetime`, when the variable in `x` is a numeric 
+#' variable, the resolution of the number (e.g. `"ms"` for milliseconds, `"s"` 
+#' for seconds, etc. link)
```

Comment:
Is this an extra word?

Reply:
Was a thing to remind me to add a link (which I missed). Done now.